### DISCUSSION TITLE
La vista tasks de una SM no se ve en dispositivos móviles?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#364] [2. Questions and Answers] [answered]
Hola

Quisiera saber o confirmar si efectivamente un usuario no pueda ver las vistas de tareas de una maquina de estado desde el celular, ya sea por vía app o del navegador del mismo móvil.

Adjunto un link de un video, entregado por un cliente indicando que tiene problemas en abrir unos links y se demora (las dos carpetas que se ven, son grupo tipo link y las redirecciono a la vista de tareas de cualquiera de los flujos). Lo probé en mi celular y efectivamente eso no sucede, e inclusive me pidió nuevamente acceso a la cuenta de cotalker. 

**Link:** https://drive.google.com/file/d/1wnfrJ9AdyrVHlyYzOqcV0qsfMmlluOK3/view?usp=sharing

Me podrían confirmar, @acedward o @guillevalin si sólo la vista de tareas se ve desde un computador y no desde un celular. Lo pregunto para cambiar la configuración o informar al cliente que sólo está disponible desde un computador.

El objetivo de la vista tasks, consiste en que los especialistas puedan realizar seguimiento y consultas de los pacientes ingresados y no buscar canal por canal o por el buscador.

Gracias =>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG por ahora solo se puede desde un computador.

Está como funcionalidad experimental (Toggle Feature), pero todavía no está liberada porque hay que trabajar harto en la experiencia de usuario dentro de un celular:
![image](https://user-images.githubusercontent.com/34345309/129221327-f3e996de-466c-4e4b-886e-c1f84d5fbe13.png)

Saludos!

**Reply by NicolasPLG** [reply]
@guillevalin aa entiendo!!, cerraré el ticket e informaré al cliente que la vista será por medio de la web.

Gracias =>
